python_coreml_stable_diffusion/unet.py

```diff
@@ -6,7 +6,7 @@
 from python_coreml_stable_diffusion.layer_norm import LayerNormANE
 
 from diffusers.configuration_utils import ConfigMixin, register_to_config
-from diffusers.modeling_utils import ModelMixin
+from diffusers.models.modeling_utils import ModelMixin
```

Comment:
```suggestion
from diffusers import ModelMixin
```

That should be enough :) (My duplicate fix is in #111).